arrows/vtk/applets/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#
+# CMake file for the arrows.mvg.applets
+#
+
+set( sources
+  register_applets.cxx
+  color_mesh.cxx
+  )
+
+set( headers
+  color_mesh.h
+  )
+
+###
+# Add our plugin path to the default list
+include_directories( ${CMAKE_CURRENT_BINARY_DIR} )
+
+# Add applet plugin
+kwiver_add_plugin( kwiver_algo_vtk_applets
+  SUBDIR       ${kwiver_plugin_applets_subdir}
+  SOURCES      ${sources} ${headers}
+  PRIVATE      kwiver_algo_core
+               kwiver_algo_vtk
+               kwiver_tools_applet
+               kwiversys
+               ${VTK_LIBRARIES}
```

Comment:
In some of the other applets I believe we link against specific vtk sub-libraries rather than all of them. This certainly seems cleaner, but I was wondering if there's a reason for the other approach?

Reply:
Probably the link command would take less time, and also its nice to see all the libraries (if there are not too many). I'll change it.